book.rb

```diff
@@ -1,10 +1,16 @@
+require_relative 'rental'
+
 class Book
   attr_accessor :title, :author
-  attr_reader :rents
+  attr_accessor :rents
 
   def initialize(title, author)
     @title = title
     @author = author
     @rents = []
   end
+
+  def add_rental(date,book)
+    Rental.new(date,self,person)
+  end
```

Comment:
- [ ] Kindly change the parameter to be `person` instead of `book` so it will be like 
```
def add_rental(date, person)
```